OpenTESArena/CMakeLists.txt

```diff
@@ -73,16 +75,38 @@ SET(TES_SOURCES
     ${TES_WORLD}
     ${TES_MAIN})
 
+SET(TES_DATA_FOLDER ${CMAKE_SOURCE_DIR}/data)
+SET(TES_OPTIONS_FOLDER ${CMAKE_SOURCE_DIR}/options)
+
 IF (WIN32)
     LIST(APPEND TES_SOURCES ${TES_RESOURCES})
     ADD_DEFINITIONS("-D_SCL_SECURE_NO_WARNINGS=1")
-ENDIF()
 
```

Comment:
I think there should be a small revision here, otherwise it is leaving out Linux.

```CMake
IF (WIN32)
    LIST(APPEND TES_SOURCES $(TES_RESOURCES))
    ADD_DEFINITIONS("-D_SCL_SECURE_NO_WARNINGS=1")
ENDIF()

IF (NOT APPLE)
    # Copy over required files...
    ...

    # Add the rest
    ...
ELSE (APPLE)
    # Info.plist properties
    ...
    ...
   # Add the rest
    ...
ENDIF()
```

Reply:
Good call. I've made the change.

Reply:
Oh, sorry, I meant to have those two lines at the top be their own if block, because with this change it will incorrectly set the Windows resources folder on Linux as well.

Reply:
I think I understood this time. Take a look and let me know.

Reply:
Yes, that looks more correct now. On another note, why were the curly braces around TES_RESOURCES changed to parentheses at `LIST(APPEND TES_SOURCES $(TES_RESOURCES))`?

Reply:
Looks like that was an oversight. Probably deleted it, then retyped it later and didn't catch it because it was under the Windows code. I've reverted it.